### ISSUE TITLE
[a11y-defect-page-title]: Unique Page Title for FSR

### ISSUE THREAD

**Issue by josephrlee** [department-of-veterans-affairs/va.gov-team#66851]
### Point of contact

Joseph Lee

### Severity level

4, Minor. Consider fixing or exploring in 2-4 sprints post-launch.

### Details

Currently, all page titles in the FSR are the same.

`<title>Financial Status Report | Veterans Affairs</title>`

To help with accessibility, each page of the FSR should have a unique page title based on the unique `h3` of every page before the form name.

`<title>{h3 from page} - Financial Status Report | Veterans Affairs</title>`

### Reproduction steps

1. Navigate through the FSR form and read out the page title in the head

### Proposed solution or next steps

Add a unique page title based on every page's unique `h3` before the form name.

`<title>{h3 from page} - Financial Status Report | Veterans Affairs</title>`

### Type of issue

- [ ] Axe-core
- [ ] Screenreader | assistive tech & device support
- [ ] Keyboard
- [ ] Zoom
- [ ] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [ ] Focus
- [ ] Headings
- [ ] Content style guide
- [ ] Link and button labels
- [ ] Semantic HTML
- [x] Markup and metadata
- [ ] Non-text content and media
- [X] Overall user experience
- [ ] Cognitive | user flows & navigating
- [ ] URL standards
- [ ] Something else

**Comment by josephrlee** [comment]
@Dr-Pongo, found this [interesting article on SPAs and page titles](https://hidde.blog/accessible-page-titles-in-a-single-page-app/). I think it will be helpful for this ticket. And here's [slack convo](https://dsva.slack.com/archives/C5HP4GN3F/p1691012295026339?thread_ts=1691001649.396199&cid=C5HP4GN3F) about it as well.

**Comment by amponce** [comment]
Update: 

@josephrlee and I discussed the changes and decided to pair up on this ticket. I have implemented the custom hook to set the document title at the app level. Joe will modify the titles in our chapter config files to be unique and match the corresponding page's H3 value.

![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/3916436/4e2f4462-8189-4177-9582-7b493c7e3a8e)

Config files to be updated:
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/3916436/145f2e51-0f88-49bd-83bf-6f881782af25)


Example change: ` title: 'Employment' would become title: 'Employment Summary',`
```js
example change
 employmentHistorySummary: {
        path: 'employment-history',
        title: 'Employment',
        uiSchema: {},
        schema: { type: 'object', properties: {} },
        depends: formData => formData.questions.vetIsEmployed,
        editModeOnReviewPage: true,
        CustomPage: EmploymentHistoryWidget,
        CustomPageReview: EmploymentHistorySummaryReview,
     }
 ```


Draft PR: [Draft PR Link](https://github.com/department-of-veterans-affairs/vets-website/pull/30216)
